### DISCUSSION TITLE
Grep?

### DISCUSSION THREAD

**Discussion by superchargez** [microsoft/sudo#51] [Ideas]
Please also give us grep

**Comment by DHowett** [comment]
Thanks for the feedback! This repository is narrowly focused on producing Sudo for Windows, so I am going to close this discussion, but I really appreciate your suggestion!

**Comment by gerardog** [comment]
Do you use scoop or winget? Try `scoop install grep` or `winget install -e --id GnuWin32.Grep`

**Reply by superchargez** [reply]
Tried winget on ffmpeg but I found winget was broken (at least for FFMPEG) have not tried it for many other things though., also, it would be nice to have something from MS directly than a third party. So, if you know their implementation of grep it would be so much helpful. Thanks.
